frontend/src/shared/modules/saved-views/components/forms/SavedViewForm.vue

```diff
@@ -0,0 +1,445 @@
+<template>
+  <app-drawer v-model="isDrawerOpen" :title="title" size="600px">
+    <template #content>
+      <div class="-mx-6 -mt-4">
+        <!-- Share with workspace -->
+        <section class="border-y border-gray-200 bg-gray-50 py-4 px-6">
+          <div class="flex justify-between items-center">
+            <div>
+              <h6 class="text-sm font-medium leading-5">
+                Shared with workspace
+              </h6>
+              <p class="text-2xs leading-4.5 text-gray-500">
+                If enabled, the view will be shared with all users from this workspace.
+              </p>
+            </div>
+            <div>
+              <el-switch v-model="form.shared" :disabled="isEdit" />
```

Comment:
Why are we disabling this in edition?

Reply:
because you cannot switch views from being tenant visible and user visible. Its how it was in designs